File: Swift/프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.swift
import Foundation

func solution(_ dots:[[Int]]) -> Int {
    let xCoordinates = dots.map{ $0[0] }
    let yCoordinates = dots.map{ $0[1] }
    
    let width = xCoordinates.max()! - xCoordinates.min()!
    let height = yCoordinates.max()! - yCoordinates.min()!
    
    return width * height
}

File: Python3/프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    x_coordinates = [dot[0] for dot in dots]
    y_coordinates = [dot[1] for dot in dots]
    
    width = max(x_coordinates) - min(x_coordinates)
    height = max(y_coordinates) - min(y_coordinates)
    
    return width * height